Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -1243,6 +1251,8 @@ final class StatusTableViewController: LoopChartsTableViewController {
                             DispatchQueue.main.async {
                                 switch result {
                                 case .success:
+                                    self.updateBannerAndHUDandStatusRows(statusRowMode: .canceledBolus(dose: dose), newSize: nil, animated: true)
+                                    self.bolusState = .noBolus
```

Comment:
Similar here as ^. Setting the `bolusState` at these known points removes flickering.